edenia/antelope-tools
# Feat/add stats stress test dashboard

### Show block history stats in the stress test dashboard

### What does this PR do?

- Show cpu and net usage percentage in the tpb and tps graphic
- Enable zoom for historical tpb and tps graphic 
- Resolve #1217 

### Steps to test

1. Run the project locally
2. Go to Stress Test page
3. Check the cards with the stats related with the block history

### Screenshot

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/f693e513-9125-4da3-8863-470f2db96f62)
